### DISCUSSION TITLE
关于音画不同步的提问

### DISCUSSION THREAD

**Discussion by xwc1234a** [hooke007/mpv_PlayKit#138] [Q&A 问答] [answered]
你好，我使用播放器播放视频时画面的播放速度总是比音频快，只有统计信息数据功能的时候画面的速度才恢复正常，播放器使用的是默认设置，也确定没有打开倍速播放，想问一下该怎么处理。

**Comment by hooke007** [comment, marked as answer]
怎么个快法？log？

**Reply by xwc1234a** [reply]
我录了一段视频播放的情况，在s/1AvE9uPAMoSleOU1IUcQ0Qg 提取码: h2ur 
另外想请教一下怎么查看log呢

**Reply by hooke007** [reply]
阅读mpv.conf
discussion支持上传视频，没有必要用其他的方式

**Reply by xwc1234a** [reply]
https://user-images.githubusercontent.com/45220968/172810114-41cd1488-8bbe-4a6d-925c-db5071048d37.mp4

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/wiki/0_FAQ#--%E4%BB%85%E7%94%BB%E9%9D%A2%E5%8A%A0%E9%80%9F

**Reply by xwc1234a** [reply]
谢谢，设置垂直同步后问题解决了
